pkgs/by-name/turbomole/package.nix

```diff
@@ -16,14 +16,21 @@
 , coreutils
 , # Configuration
   useMPI ? false
+, version ? "7.8.1"
```

Comment:
Thanks!
Now you do not have version 7.9 in the set (i.e. `turbomole` also evaluates to 7.8.1).

Reply:
Ah oh good point :sweat_smile: Fixed.